uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
```

Comment:
This is now done.  If a plugin wanted to keep donation tiers turned on for ATD reg, they could just add this line to their ``regextras.html`` file:

```javascript
window.removeDonationRows = $.noop;
```

which replaces the function doing the removing with a noop function.